retention updates (#958)
timescale/docs: timescaledb/getting-started/data-retention.md

# Create a data retention policy

An intrinsic part of working with time-series data is that the relevance of
data can diminish over time. New data is accumulated and old data
is rarely, if ever, updated. It is therefore often desirable to delete old raw
data to save disk space.

> In practice, data is often downsampled first, for example through continuous aggregates, 

Reviewer: ```suggestion
In practice, continuous aggregates downsample the data, and then
```

> then historic raw data points are discarded via data retention policies.

Reviewer: ```suggestion
 data retention policies discard historic raw data points.
```

<highlight type="note">
Notice in the following image that dropping data on the underlaying hypertable does not 
affect the continuous aggregate. As long as you do not refresh the continuous aggregate 

> for time periods that you dropped data, your aggregate will be unaffected.

Reviewer: ```suggestion
for time periods that you dropped data, your aggregate is unaffected.
```
